src/query/api/v1/handler/graphite/find_parser.go

```diff
@@ -24,71 +24,56 @@ import (
 	"fmt"
 	"io"
 	"net/http"
-	"time"
 
 	"github.com/m3db/m3/src/query/errors"
-	"github.com/m3db/m3/src/query/graphite/graphite"
 	graphiteStorage "github.com/m3db/m3/src/query/graphite/storage"
+	"github.com/m3db/m3/src/query/models"
 	"github.com/m3db/m3/src/query/storage"
 	"github.com/m3db/m3/src/query/util/json"
 	"github.com/m3db/m3/src/x/net/http"
 )
 
-func parseFindParamsToQuery(r *http.Request) (
-	*storage.FetchQuery,
+func parseFindParamsToQueries(r *http.Request) (
+	*storage.CompleteTagsQuery,
+	*storage.CompleteTagsQuery,
+	string,
 	*xhttp.ParseError,
 ) {
 	values := r.URL.Query()
-	now := time.Now()
-	fromString, untilString := r.FormValue("from"), r.FormValue("until")
```

Comment:
Why are we getting rid of the from and until? I thought the new index supports that.

Reply:
Good one, for some reason decided these are *always* from `time.Time{}` until `time.Now()`, rather than by default (I think because we drop em later which was wrong too)